packages/core/src/components/index.ts

```diff
@@ -0,0 +1,25 @@
+import type { JSX as IonicJSX } from '@ionic/core/dist/types/components';
+import type { ParentProps, JSX as JSXBase } from 'solid-js';
+import { FixVanillaIonProps } from '../lib';
+
```

Comment:
I think Biome was complaining on this line :grinning:
```suggestion
import type { FixVanillaIonProps } from '../lib';
```

Reply:
vscode's PR review plugin (which I had to use because github crashes due to the PR size) is apparently buggy. The suggestion is bad, don't use it directly :sweat_smile: